City-of-Pori/pori-kada
# Added multiple module updates, updated core to 7.55, recreated blog f…

…eature

To test:

1. build the site
2. drush fra -y && drush cc all
3. Make sure features work about correctly.
